src/index.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+import SQLModule from './sql/sql';
+import ConnectionOptions from './connectionOptions';
+import Constants from './constants';
+import { getCookieValue, isMindsDbCloudEndpoint } from './util/http';
+
+const defaultAxiosInstance = axios.create({
+  baseURL: Constants.BASE_CLOUD_API_ENDPOINT,
+});
+
+const SQL = new SQLModule.SqlRestApiClient(defaultAxiosInstance);
+
+/**
+ * Initializes the MindsDB SDK and authenticates the user if needed.
+ * @param {ConnectionOptions} options - Options to use for initialization
+ */
+const connect = async function (options: ConnectionOptions): Promise<void> {
```

Comment:
I have faced the following warning in the ts linter:

```
An async function or method in ES5/ES3 requires the 'Promise' constructor
```
Checking a good option to fix this, adding "es2015.promise" in tsconfig.json - in "lib" = like this :

 **tsconfig.json**
```
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "declaration": true,
    "outDir": "dist",
    "strict": true,
     "lib": [
            "es2015"
        ],
  },
  "include": ["src"],
  "exclude": ["node_modules", "dist", "**/__tests__/*"]
}

```